app/templates/admin/users/list.hbs

```diff
@@ -0,0 +1,85 @@
+<div class="sixteen wide column">
+  <table class="ui compact stackable very basic table">
+    <thead>
+      <tr>
+        <th>{{t 'Name'}}</th>
+        <th>{{t 'Email'}}</th>
+        <th>{{t 'Status'}}</th>
+        <th>{{t 'System Roles'}}</th>
+        <th>{{t 'Event Roles'}}</th>
+        <th>{{t 'User Links'}}</th>
+        <th>{{t 'Member Since'}}</th>
+        <th>{{t 'Last Access'}}</th>
+        <th>{{t 'Options'}}</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{#each model as |user|}}
+        <tr>
+          <td>
+            {{user.name}}
+          </td>
+          <td>
+            {{user.email}}
+          </td>
+          <td>
+            {{#if (eq user.status 'active')}}
+              <div class="ui green label">{{t 'Active'}}</div>
+            {{else}}
+              <div class="ui red label">{{t 'Inactive'}}</div>
+            {{/if}}
+          </td>
+          <td>
+            <div class="ui ordered list">
+              {{#each user.systemRoles as |systemRoles|}}
+                <div class="item">{{systemRoles}}</div>
+              {{/each}}
+            </div>
+          </td>
+          <td>
+            <div class="ui ordered list">
+              {{#each user.eventRoles as |eventRoles|}}
+                <div class="item">{{eventRoles.name}} ({{eventRoles.event.name}})</div>
```

Comment:
make this `as |eventRole|` instead of `as |eventRoles|`, makes more sense that way